keyboards/manta60/info.json

```diff
@@ -0,0 +1,81 @@
+              {"label":"@", "x":2, "y":0},
+              {"label":"#", "x":3, "y":0},
+              {"label":"$", "x":4, "y":0},
+              {"label":"%", "x":5, "y":0},
+              {"label":"=", "x":6, "y":0},
+              {"label":"-", "x":10, "y":0},
+              {"label":"^", "x":11, "y":0},
+              {"label":"&", "x":12, "y":0},
+              {"label":"*", "x":13, "y":0},
+              {"label":"(", "x":14, "y":0},
+              {"label":")", "x":15, "y":0},
+              {"label":"\\", "x":16, "y":0},
+              {"label":"-", "x":0, "y":1},
+              {"label":"Q", "x":1, "y":1},
+              {"label":"W", "x":2, "y":1},
+              {"label":"E", "x":3, "y":1},
+              {"label":"R", "x":4, "y":1},
+              {"label":"T", "x":5, "y":1},
+              {"label":"[", "x":6, "y":1},
+              {"label":"]", "x":10, "y":1},
+              {"label":"Y", "x":11, "y":1},
+              {"label":"U", "x":12, "y":1},
+              {"label":"I", "x":13, "y":1},
+              {"label":"O", "x":14, "y":1},
+              {"label":"P", "x":15, "y":1},
+              {"label":"@", "x":16, "y":1},
+              {"label":"Tab", "x":0, "y":2},
+              {"label":"A", "x":1, "y":2},
+              {"label":"S", "x":2, "y":2},
+              {"label":"D", "x":3, "y":2},
+              {"label":"F", "x":4, "y":2},
+              {"label":"G", "x":5, "y":2},
+              {"label":",", "x":6, "y":2},
+              {"label":".", "x":10, "y":2},
+              {"label":"H", "x":11, "y":2},
+              {"label":"J", "x":12, "y":2},
+              {"label":"K", "x":13, "y":2},
+              {"label":"L", "x":14, "y":2},
+              {"label":"UP", "x":15, "y":2},
+              {"label":";", "x":16, "y":2},
+              {"label":"Shft", "x":0, "y":3},
+              {"label":"Z", "x":1, "y":3},
+              {"label":"X", "x":2, "y":3},
+              {"label":"C", "x":3, "y":3},
+              {"label":"V", "x":4, "y":3},
+              {"label":"Alt", "x":5, "y":3},
+              {"label":"B", "x":11, "y":3},
+              {"label":"N", "x":12, "y":3},
+              {"label":"M", "x":13, "y":3},
+              {"label":"LEFT", "x":14, "y":3},
+              {"label":"DOWN", "x":15, "y":3},
+              {"label":"RIGHT", "x":16, "y":3},
+              {"x":5, "y":4.5},
+              {"x":11, "y":4.5, "h":2},
+              {"label":"Lower", "x":0, "y":5.5},
+              {"x":1, "y":5.5},
+              {"label":"Alt", "x":3, "y":5.5},
+              {"label":"Ctrl", "x":4, "y":5.5},
+              {"label":"Enter", "x":6, "y":5.5},
+              {"label":"cmd", "x":7, "y":5.5},
+              {"label":"Del", "x":9, "y":5.5},
+              {"label":"BS", "x":10, "y":5.5},
+              {"label":"Ctrl", "x":12, "y":5.5},
+              {"label":"Alt", "x":13, "y":5.5},
+              {"x":15, "y":5.5},
+              {"label":"Rize", "x":16, "y":5.5}
```

Comment:
Lines 64-77:

The objects here are out of order with respect to the code in `manta60.h`, but it'll actually be easier for me to fix it later than to give you a suggestion here at this time.